src/renderer/components/map/MapComponent.js

```diff
@@ -257,22 +256,24 @@ class MapComponent extends React.Component {
     })
     if (contactFeature) {
       if (contactFeature.properties.msgId) {
-        const messageObj = await ipcRenderer.sendSync(
```

Comment:
I guess switching to callDcMethod allows us to delete the old ipc event?

Reply:
which "old ipc event" do you mean? This one:
https://github.com/deltachat/deltachat-desktop/blob/master/src/main/ipc.js#L106
that wasn't called here before this change...?
But seems to be obsolete anyway...